app/models/ability.rb

```diff
@@ -23,8 +23,8 @@ def not_signed_in
       conference.splashpage&.public == true
     end
     # Can view the schedule
-    can [:schedule, :events], Conference do |conference|
-      conference.program.cfp && conference.program.schedule_public
+    can [:show, :events], Schedule do |schedule|
```

Comment:
:schedule and :show are two different [abilities](https://github.com/CanCanCommunity/cancancan/wiki/Custom-Actions). both are needed.

Reply:
why? there is not `schedule` method in the (conference controller)[https://github.com/openSUSE/osem/blob/master/app/controllers/conferences_controller.rb] :confused: